jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -58,8 +58,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <goto-analyzer/unreachable_instructions.h>
 
 janalyzer_parse_optionst::janalyzer_parse_optionst(int argc, const char **argv)
-  : parse_options_baset(JANALYZER_OPTIONS, argc, argv),
-    messaget(ui_message_handler),
+  : messaget(ui_message_handler),
+    parse_options_baset(JANALYZER_OPTIONS, argc, argv, *this),
```

Comment:
Here and everywhere else: what about only passing the message _handler_, not the full `messaget`? I think that would reduce the amount of churn in this commit, and would still achieve the same effect.

Reply:
Can do...